Description: Fix build failure with glib 2.32
 where including invidual glib headers is no longer allowed.
Author: Michael Biebl <biebl@debian.org>
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=665545
Index: gtk2-engines-2.20.2/engines/clearlooks/src/animation.c
===================================================================
--- gtk2-engines-2.20.2.orig/engines/clearlooks/src/animation.c	2010-09-23 15:40:24.000000000 +0200
+++ gtk2-engines-2.20.2/engines/clearlooks/src/animation.c	2012-03-25 23:41:15.810865702 +0200
@@ -28,7 +28,7 @@
 #include "animation.h"
 
 #ifdef HAVE_WORKING_ANIMATION
-#include <glib/gtimer.h>
+#include <glib.h>
 
 struct _AnimationInfo {
 	GTimer *timer;
